pkgs/top-level/all-packages.nix

```
@@ -28195,7 +28195,9 @@ with pkgs;
 
   # zen-kernel
   linuxPackages_zen = linuxKernel.packages.linux_zen;
+  linux_zen = linuxPackages_zen.kernel;
```

I'd prefer to keep it in line with the other entries.

```suggestion
  linux_zen = linuxKernel.kernels.linux_zen
```

---

It's pretty inconsistent, the other kernels have a mix of both.